core/src/main/java/hudson/tasks/BatchFile.java

```diff
@@ -72,8 +93,28 @@ public String getDisplayName() {
         }
 
         @Override
-        public Builder newInstance(StaplerRequest req, JSONObject data) {
-            return new BatchFile(data.getString("command"));
+        public Builder newInstance(StaplerRequest req, JSONObject formData) {
+            return req.bindJSON(BatchFile.class,formData);
```

Comment:
This is the FindBugs issue since `req` is [CheckForNull](https://github.com/jenkinsci/jenkins/blob/96ec7a267e0efba2ec99590c871db0940e35920f/core/src/main/java/hudson/model/Descriptor.java#L547...L552).

Still not clear to me why this is even overridden. Doesn't the default `Descriptor#newInstance(…)` do the exact same thing?